src/streamlink/plugins/nrk.py

```diff
@@ -3,30 +3,10 @@
 
 from streamlink.plugin import Plugin
 from streamlink.plugin.api import validate
-from streamlink.stream import HLSStream
+from streamlink.stream import HLSStream, HTTPStream
 
-COOKIE_PARAMS = (
-    "devicetype=desktop&"
-    "preferred-player-odm=hlslink&"
-    "preferred-player-live=hlslink"
-)
-
-_id_re = re.compile(r"/(?:program|direkte|serie/[^/]+)/([^/]+)")
+_id_re = re.compile(r"/(program|direkte|serie|podkast)(?:/.+)?/([^/]+)")
```

Comment:
Could you please move the regexes and the schemas into the class as class attributes? The current plugin implementation is still using the old layout which we don't want anymore.